data/settings.js

```diff
@@ -9474,9 +9474,13 @@ If yes, import all fields returned by the passdb lookup. If no, require
 		tags: [ 'passdb' ],
 		values: setting_types.STRING,
 		default: {
-			value: 'PLAIN',
+			value: 'CRYPT',
```

Comment:
you sure about this? isn't it only for passwd and passwd-file?

Reply:
Although looking at the change, it changes the global default_password_scheme to CRYPT. And then all of the defaults are CRYPT. So this part of the change is correct.